allen-cell-animated/nucmorph-colorizer
# fix: Adjusted tooltip colors

Problem
=======
Closes #236, "Update tooltip background colors."

*Estimated review size: tiny, 2 minutes*

Solution
========
- Updates tooltip background to a transparent mid grey to match suggested designs.

## Type of change

* New feature (non-breaking change which adds functionality)

Steps to Verify: (optional)
----------------
1. Open preview link: https://allen-cell-animated.github.io/nucmorph-colorizer/pr-preview/pr-298/#/
2. Click around the UI to see the new tooltips.

Screenshots (optional):
-----------------------
| Before | After |
| --- | --- |
| ![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/58a6aa66-5d57-4887-84a7-429db4c087bc) | ![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/2b822073-00c1-4247-a9ea-a9564db8b880) |

![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/0eb2e1c8-ece3-4de4-97e1-520c68c96ccb)
